build_tools/cmake/build_tf_binaries.sh

```diff
@@ -1,74 +0,0 @@
-#!/bin/bash
-
-# Copyright 2022 The IREE Authors
-#
-# Licensed under the Apache License v2.0 with LLVM Exceptions.
-# See https://llvm.org/LICENSE.txt for license information.
-# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-
-# Build the IREE TF integration binaries. Despite the name, also runs the few
-# lit tests for these that are enabled through Bazel. These take seconds to run
-# all of them and it's easiest to just run them here rather than trying to
-# figure out how to pass prebuilt binaries to Bazel tests.
```

Comment:
So as noted here, this was also running all the lit tests under this directory. If those are unnecessary, let's delete them now.

Reply:
They would be necessary for bazel testing I believe - no compilation, but else cmake & lit is only test. But there is nothing bazel build specific, its just bazel invoking lit really.

Reply:
Done